src/tools/kdb/import.cpp

```diff
@@ -75,6 +75,12 @@ int ImportCommand::execute (Cmdline const & cl)
 
 	if (cl.strategy == "validate")
 	{
+		if (cl.ns.empty ())
```

Comment:
Also here it is not specific to validate: `kdb import` does not have a way to import without namespace.

Reply:
Same reasoning as with `kdb editor` (`/doc/help/kdb-import.md`).

Reply:
I think the docu is wrong, at least for import I am quite sure cascading imports never work (regardless of strategy). It is more safe if we disable it (for import+editor) and reintroduce it later if there is proper support for it. (unless you tested it and it works)

Can you please also add tests in which ambiguous cases are triggered (and `kdb` fails).

Reply:
Both are disabled as of now; tests with such cases are now found in `cascading.md`.